src/sdk/namespace_fs.js

```diff
@@ -756,7 +756,7 @@ class NamespaceFS {
                         };
                     } else {
                         r = {
-                            key: this._get_entry_key(dir_key, ent, isDir),
+                            key: this._get_entry_key(dir_key, ent, isDir, is_disabled_dir_content),
```

Comment:
this handles versioning disabled, where is the change for list_versions? do we need to have in _get_version_entry_key an extended implementation?

Reply:
the change is inside the function. essentially there is no special case for the extended implementation. we use the .folder file as if it is a regular key

Reply:
the solution is in _get_version_entry_key. removing the line so the object is processed. we format the name in the format key function at the end of list-objects